int8 ua_type
string uas_id
string uas_caa_id
string description
string operator_id
float64 latitude
float64 longitude
float64 altitude
float64 altitudegeo
float64 altitudebaro
float32 speed_horizontal
float32 speed_vertical
float64 height
float32 direction
int8 height_type
int8 horiz_accuracy
int8 vert_accuracy
int8 speed_accuracy